src/generator/AutoRest.Ruby.Azure/TemplateModels/AzureMethodTemplateModel.cs

```diff
@@ -119,7 +119,7 @@ public override string ResponseGeneration()
                 if (ReturnType.Body is CompositeType)
                 {
                     CompositeType compositeType = (CompositeType)ReturnType.Body;
-                    if (compositeType.Extensions.ContainsKey(AzureExtensions.PageableExtension))
+                    if (compositeType.Extensions.ContainsKey(AzureExtensions.PageableExtension) && this.Extensions.ContainsKey("nextMethodName"))
```

Comment:
similar to the other comment, "nextMethodName" is different form "nextLinkMethod"

---
In reply to: [74298007](https://github.com/Azure/autorest/pull/1335#discussion_r74298007) [](ancestors = 74298007)